cpp/src/Slice/Parser.h

```diff
@@ -280,6 +280,8 @@ namespace Slice
 
         /// Contains all introductory lines up to the first tag.
         [[nodiscard]] StringList overview() const;
+        /// Returns the contents of any '@remarks' tags.
```

Comment:
```suggestion
        /// Returns the contents of any '@remarks' or '@remark' tags.
```

Reply:
Fixed, but I swapped the order.
If we list both, I think `remark` should come before `remarks`.